test/gtest/uct/ib/test_dc.cc

```diff
@@ -345,6 +356,10 @@ UCS_TEST_P(test_dc, dcs_ep_purge_pending, "DC_NUM_DCI=1") {
     m_e1->connect_to_iface(0, *m_e2);
 
     iface = dc_iface(m_e1);
+    if (uct_dc_mlx5_iface_is_hybrid(iface)) {
+        UCS_TEST_SKIP_R("hybrid DCI policy does not support pending purge requests while ndci = 1");
```

Comment:
ditto

Reply:
The test assumes that we only have a single DCI when setting DC_NUM_DCI=1 but in hybrid mode we also have the HW DCI which will prevent out-of-resource status